app/pb_kits/playbook/pb_person_contact/docs/example.yml

```diff
@@ -4,6 +4,7 @@ examples:
   - person_contact_default: Default
   - person_contact_multiple: Multiple People
   - person_contact_with_detail: With Detail
+  - person_contact_with_wrong_numbers: With Wrong Numbers
```

Comment:
Do we need all those extra lines right above this one?

Reply:
Line 7 adds the documentation to the Playbook UI for the example with the wrong numbers in the contact. Lines 4, 5 & 6 are different examples, showing that there are different options/props you can use with the kit.

Reply:
Wow, sorry I thought about something but typed other. I meant "the lines below".

Reply:
Not sure why the multiple spaces... looks like there isn't a solid convention used throughout to separate. I'm a fan of a single space, which is what I'll use here. FWIW, many kits have 1-2 spaces between, without a clear reason why.